ensawards.org/src/data/bestPractices.ts

```diff
@@ -75,29 +76,25 @@ export const displayNamedSmartContracts: BestPractice = {
   slug: "display-named-smart-contracts",
   name: "Display named smart contracts",
   description:
-    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    "Display ENS names instead of addresses when users interact with named smart contracts.",
   categoryName: "Contract naming",
   categorySlug: "contract-naming",
   appliesTo: [BestPracticeApplications.App],
   technicalDetails: {
     main: {
       header: "Technical Details",
       content:
-        "The following is temporary placeholder content. Technical details will be added soon. " +
-        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. " +
-        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. " +
-        "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. " +
-        "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+        "Looking up the name of a smart contract uses the same process as looking up the name of any other account. " +
+        "You can use a variety of libraries and tools such as [Viem](https://viem.sh/docs/ens/actions/getEnsName) or [ensjs](https://github.com/ensdomains/ensjs/blob/main/docs/public/function.getName.md) which can perform a reverse lookup of a [primary name](https://docs.ens.domains/web/reverse) using the contract address. " +
```

Comment:
```suggestion
        "You can use a variety of libraries to lookup the [primary name](https://docs.ens.domains/web/reverse) of a contract address such as [Viem](https://viem.sh/docs/ens/actions/getEnsName), [ensjs](https://github.com/ensdomains/ensjs/blob/main/docs/public/function.getName.md), or ensnode-sdk." +
```

We should promote the use of the ENSNode client in ENSNode-sdk. In other words, we want to promote how ENSNode can be used through a simple library integration, similar to viem. This is important for promoting ENSNode adoption.

Please also include a link here for making use of `ensnode-sdk`.